src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -161,7 +161,7 @@ define(function (require, exports, module) {
         // transport, check message before sending to remote, and reload if necessary
         // without actually sending to remote for processing.
         if(msg.method === "Page.reload" || msg.method === "Page.navigate") {
-            reload();
+            reload(true);
```

Comment:
We should force reload (i.e. send `true` to the `reload` function) only for `Page.navigate` not for `Page.reload`